src/amo/components/AddonDetail.js

```diff
@@ -28,13 +58,13 @@ export class AddonDetail extends React.Component {
             <LikeButton />
           </div>
           <div className="title">
-            <h1>Placeholder Add-on Title
-              <span className="author">by <a href="#">AwesomeAddons</a></span></h1>
-            <InstallButton slug="placeholder" />
+            <h1>
+              {addon.name} <span className="author">{i18n.gettext('by')} {authorList}</span>
```

Comment:
@muffinresearch I almost forgot to localize this. Is it ok to isolate the `by` like this? It seems like creating a longer format string would be complicated because of the span tag.